src/main/java/mate/academy/service/UserService.java

```diff
@@ -10,11 +10,28 @@ public class UserService {
 
     /**
      * Find user by email. All users are stored in <code>private static final User[] users</code>
+     *
      * @param email - the input parameter
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
-        return null;
+    public boolean findByEmail(String email) {
+        boolean equalEmail = false;
```

Comment:
we can avoid creating redundant variable and return user if user.getEmail() is equal to the one which is passed as a parameter